pkg/unikontainers/unikernels/rumprun.go

```diff
@@ -58,27 +75,37 @@ func (r RumprunConfig) ToJSONString() (string, error) {
 	return jsonStr, nil
 }
 
-func NewRumprunConfig(data ExecData) (RumprunConfig, error) {
+func newRumprun(data UnikernelParams) (Rumprun, error) {
 	tempBlk := RumprunBlk{
 		Source:     "etfs",
 		Path:       "/dev/ld0a",
 		FsType:     "blk",
 		Mountpoint: "/data",
 	}
-	mask, err := unet.SubnetMaskToCIDR(data.Network.EthDevice.Mask)
+
+	// if EthDeviceMask is empty, there is no network support
+	if data.EthDeviceMask == "" {
+		return Rumprun{
+			Command: data.CmdLine,
+			Net:     RumprunNet{Mask: ""},
+			Blk:     tempBlk,
+		}, nil
+	}
+
+	mask, err := subnetMaskToCIDR(data.EthDeviceMask)
```

Comment:
add the following comment:
`FIXME: in the case of rumprun & k8s, we need to explicitly set the mask to an inclusive value (eg 1 or 0), as NetBSD complains and does not set the default gw if it is not reachable from the IP address directly.`

Reply:
Added